src/lib/solana/manuallyExecuteSolana.ts

```diff
@@ -114,3 +146,116 @@ export function newAnchorProvider(chainName: string) {
   })
   return { anchorProvider, keypair }
 }
+
+type ManualExecAccounts = {
+  config: PublicKey
+  referenceAddresses: PublicKey
+  sourceChain: PublicKey
+  commitReport: PublicKey
+  offramp: PublicKey
+  allowedOfframp: PublicKey
+  rmnRemote: PublicKey
+  rmnRemoteCurses: PublicKey
+  rmnRemoteConfig: PublicKey
+  authority: PublicKey
+  systemProgram: PublicKey
+  sysvarInstructions: PublicKey
+}
+
+async function bufferedTransactions(
+  destinationProvider: AnchorProvider,
+  bufferProgramAddress: string,
+  serializedReport: Buffer,
+  serializedTokenIndexes: Buffer<ArrayBuffer>,
+  originalManualExecAccounts: ManualExecAccounts,
+  originalManualExecRemainingAccounts: AccountMeta[],
+  computeBudgetIx: TransactionInstruction,
+  blockhash: string,
+  addressLookupTableAccounts: AddressLookupTableAccount[],
+): Promise<VersionedTransaction[]> {
+  // Arbitrary as long as there's consistency for all translations.
+  const bufferId = {
+    bytes: Array.from(randomBytes(32)),
```

Comment:
As discussed offline, please either:
- Log the used ID
- Provide a CLI command to close a buffer (in which case you could even just use the message ID as the buffer ID, and just close it if it failed in the middle)